docs/3.api/3.utils/clear-nuxt-data.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'clearNuxtData'
-description: Delete cached data, error status and pending promises of useAsyncData and useFetch.
+description: Delete cached data and error status promises of useAsyncData and useFetch.
```

Comment:
I think this might be left unchanged, as it refers to pending _promises_.

Reply:
Yeah, i think i misunderstood the part.